ckanext/geocat/utils/xpath_utils.py

```diff
@@ -309,6 +309,9 @@ def _set_distribution_format_and_media_type(
             distribution['media_type'] = ""
     elif normed_protocol == DOWNLOAD_PROTOCOL:
         distribution['media_type'] = ""
+    elif normed_protocol in LINKED_DATA_PROTOCOL:
+        format = re.findall(r'([^:]+$)', protocol)[0]
```

Comment:
If we get the protocol `LINKED:DATA` with no format at the end, this will set the format to `DATA`. This doesn't seem correct. I think we should check that there is a format at the end of the protocol before setting this, possibly like in https://github.com/opendata-swiss/ckanext-geocat/pull/113/files#diff-6ef6edec764727c79fc5548a55a2c070e71f3523b46dd8f54d8b24b2929a3bc9R300